lib/config/validation.ts

```diff
@@ -128,19 +128,6 @@ export async function validateConfig(
       });
       continue;
     }
-    if (key === 'gitTimeout') {
-      if (!is.number(val)) {
```

Comment:
shouldn't we leave the number check?

Reply:
I thought we automatically check such types. I'll verify manually

Reply:
yeah, we should validate by default. 🙃

Reply:
https://github.com/renovatebot/renovate/pull/15206